zeek/compiler/glue-compiler.cc

```diff
@@ -235,6 +235,73 @@ GlueCompiler::GlueCompiler(Driver* driver) : _driver(driver) {}
 
 GlueCompiler::~GlueCompiler() {}
 
+hilti::Result<std::string> GlueCompiler::getNextEvtBlock(std::istream& in, int* lineno) {
```

Comment:
Can be `const` (also in decl).
```suggestion
hilti::Result<std::string> GlueCompiler::getNextEvtBlock(std::istream& in, int* lineno) const {
```